### ISSUE TITLE
[Forge] Pehkui wont work on servers

### ISSUE THREAD

**Issue by eirikangry** [Virtuoel/Pehkui#87]
I've tried to run the Forge version of the Pehkui mod on both SMpicnic and Aternos and i run into the same issue. When i try to join the server i get the message "Mismatched mod channel list". When i try to join the server without the Pehkui mod loaded the server crashes and i get the following crash report:

```
---- Minecraft Crash Report ----

// Don't be sad. I'll do better next time, I promise!

Time: 4/23/21 11:12 AM
Description: Exception ticking world

java.lang.ExceptionInInitializerError: null
	at virtuoel.pehkui.util.ScaleUtils.syncScales(ScaleUtils.java:170) ~[pehkui:2.0.0+1.16.5-forge] {re:mixin,re:classloading}
	at virtuoel.pehkui.util.ScaleUtils.syncScalesIfNeeded(ScaleUtils.java:153) ~[pehkui:2.0.0+1.16.5-forge] {re:mixin,re:classloading}
	at net.minecraft.world.TrackedEntity.handler$zzk000$onTick(TrackedEntity.java:528) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:pehkui.mixins.json:EntityTrackerEntryMixin,pl:mixin:A}
	at net.minecraft.world.TrackedEntity.func_219453_a(TrackedEntity.java:184) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:pehkui.mixins.json:EntityTrackerEntryMixin,pl:mixin:A}
	at net.minecraft.world.server.ChunkManager.func_219169_g(ChunkManager.java:1008) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerChunkProvider.func_217220_m(ServerChunkProvider.java:376) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerChunkProvider.func_217207_a(ServerChunkProvider.java:321) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:333) ~[?:?] {re:classloading,pl:runtimedistcleaner:A,re:mixin,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_282] {}
Caused by: java.lang.IllegalArgumentException: Registration of network channels is locked
	at net.minecraftforge.fml.network.NetworkRegistry.createInstance(NetworkRegistry.java:150) ~[forge:?] {re:classloading}
	at net.minecraftforge.fml.network.NetworkRegistry.newSimpleChannel(NetworkRegistry.java:117) ~[forge:?] {re:classloading}
	at virtuoel.pehkui.network.PehkuiPacketHandler.<clinit>(PehkuiPacketHandler.java:10) ~[pehkui:2.0.0+1.16.5-forge] {re:classloading}
	... 14 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at virtuoel.pehkui.util.ScaleUtils.syncScales(ScaleUtils.java:170) ~[pehkui:2.0.0+1.16.5-forge] {re:mixin,re:classloading}
	at virtuoel.pehkui.util.ScaleUtils.syncScalesIfNeeded(ScaleUtils.java:153) ~[pehkui:2.0.0+1.16.5-forge] {re:mixin,re:classloading}
	at net.minecraft.world.TrackedEntity.handler$zzk000$onTick(TrackedEntity.java:528) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:pehkui.mixins.json:EntityTrackerEntryMixin,pl:mixin:A}
	at net.minecraft.world.TrackedEntity.func_219453_a(TrackedEntity.java:184) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:pehkui.mixins.json:EntityTrackerEntryMixin,pl:mixin:A}
	at net.minecraft.world.server.ChunkManager.func_219169_g(ChunkManager.java:1008) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerChunkProvider.func_217220_m(ServerChunkProvider.java:376) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerChunkProvider.func_217207_a(ServerChunkProvider.java:321) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:333) ~[?:?] {re:classloading,pl:runtimedistcleaner:A,re:mixin,pl:runtimedistcleaner:A}
-- Affected level --
Details:
	All players: 1 total; [ServerPlayerEntity['Dabluehaired'/134, l='ServerLevel[world]', x=127.50, y=64.00, z=-233.50]]
	Chunk stats: ServerChunkCache: 2025
	Level dimension: minecraft:overworld
	Level spawn location: World: (128,64,-240), Chunk: (at 0,4,0 in 8,-15; contains blocks 128,0,-240 to 143,255,-225), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 1856 game time, 1856 day time
	Level name: world
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 122865 (now: false), thunder time: 35909 (now: false)
	Known server brands: forge
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_282] {}


-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Linux (amd64) version 5.4.0-65-generic
	Java Version: 1.8.0_282, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 852167568 bytes (812 MB) / 1542455296 bytes (1471 MB) up to 1771044864 bytes (1689 MB)
	CPUs: 3
	JVM Flags: 3 total; -XX:MaxMetaspaceSize=200M -Xmx1900M -Xms950M
	ModLauncher: 8.0.9+86+master.3cf110c
	ModLauncher launch target: fmlserver
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-4.0.0.jar eventbus PLUGINSERVICE 
		/forge.jar object_holder_definalize PLUGINSERVICE 
		/forge.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-3.0.1.jar accesstransformer PLUGINSERVICE 
		/forge.jar capability_inject_definalize PLUGINSERVICE 
		/forge.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge.jar fml TRANSFORMATIONSERVICE 
	FML: 36.1
	Forge: net.minecraftforge:36.1.0
	FML Language Providers: 
		javafml@36.1
		minecraft@1
	Mod List: 
		forge-1.16.5-36.1.0-server.jar                    |Minecraft                     |minecraft                     |1.16.5              |DONE      |NOSIGNATURE
		Pehkui-2.0.0+1.16.5-forge.jar                     |Pehkui                        |pehkui                        |2.0.0+1.16.5-forge  |DONE      |NOSIGNATURE
		forge-1.16.5-36.1.0-universal.jar                 |Forge                         |forge                         |36.1.0              |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
	Crash Report UUID: e10dcd57-024c-4537-8a9f-d9cfce6cd53b
	Player Count: 1 / 10; [ServerPlayerEntity['Dabluehaired'/134, l='ServerLevel[world]', x=127.50, y=64.00, z=-233.50]]
	Data Packs: vanilla, mod:pehkui, mod:forge
	Is Modded: Definitely; Server brand changed to 'forge'
	Type: Dedicated Server (map_server.txt)
```

**Comment by Caleb-JS** [comment]
I have the same issue. My server crashes every time someone tries to join, and says mismatched mod list.

**Comment by DepeszHubert** [comment]
Same.

**Comment by Danksukii** [comment]
Same as well
which sucks cuz I need it for my server Dx
hopefully It'll get fixed soon!

**Comment by ArtooFar** [comment]
Same.

**Comment by Virtuoel** [comment]
Resolved in 747b7e4931f15b0db30a2bbfb53502179c682b4c
New version with the fix shall be released soon.
